### DISCUSSION TITLE
New Podcaster here - added the prefix but no feed

### DISCUSSION THREAD

**Discussion by NewbiePodcaster** [skymethod/op3#34] [Q&A]
Hi, I need help. I noticed someone had a similar issue but not sure if it's the same issue. Launched my podcast 3 weeks ago, asked the host if the prefix was added correctly and they have confirmed it was. 
Here is my 1st episode -https://the-kadara-woman-podcast.simplecast.com/episodes/unlocking-the-power-of-purposeful-living-insights-from-yetunde-shorters
Thanks.

**Comment by johnspurlock-skymethod** [comment]
Hi there - I just looked into this and for some reason your show was not coming up in the search over on the [setup page](https://op3.dev/setup).  The setup page is the best way to verify your OP3 setup and find a link to the corresponding stats page.

The underlying podcast index that search uses did not know about your show, which is odd - since your show is in Apple Podcasts.  In any case, I was able to get your show added there this morning, and now OP3 knows about it as well. 

If you search for your show in the setup page, it will appear now, with a link to a blank stats page.  I'll rerun your show today to populate for the unallocated downloads OP3 has already measured.  Don't worry, as long as the prefix was present in your RSS feed (it is!) no stats are lost, just might be temporarily unallocated as was the case here.  Internally, I see downloads for this show going back to August 7th - does that roughly match up with when you added the OP3 prefix?

I'll let you know when everything is up to date on the public stats page, thanks for reaching out!

**Reply by johnspurlock-skymethod** [reply]
Alright, should be all set: your stats page is here -> https://op3.dev/show/75c89d61-5048-525f-a82c-effd84045aa8

(also available via search in the setup page)

**Comment by NewbiePodcaster** [comment]
Thank you John for the quick review.

The data is a bit skewed, is that because of the start date when I added
the RSS feed?

The number of downloads is much higher on Simplecast.

Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
Possibly - numbers reported by different systems will always be somewhat different for a variety of reasons.  Even though most systems use the IAB standard, there is room within that standard for deviation depending on what inputs are used. Also prefixed-based system like OP3 see requests somewhat differently than the actual podcast hosting company.

One of the nice things about OP3 is that it provides a standard apples-to-apples comparison for shows _across_ hosts, and consistent when a given show changes podcast hosting company, which happens surprisingly often.

Worth keeping an eye on it over the next few months and see where things land.

**Comment by NewbiePodcaster** [comment]
Thank you John.

That makes a ton of sense - one more question please.  Will this also show
top African Regions or is that a future update? I see EU, Asian regions.
However, Nigeria is in the country data.

Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
Let me go back and take a look at that spot in the system, it's been a while since the initial launch.  If there isn't already a box for detailed regions in Africa, it should be easy to add given the underlying data.

**Comment by NewbiePodcaster** [comment]
Hi John,

Thank you- yes, if you can please add the Africa box, that would be
helpful. My guests are from Africa so it will help to see if we have any
traffic from the continent too.

Thank you again. Have a good weekend ahead.

Lola

Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
Ok, the Top African regions box is now appearing for your show...
<img width="379" alt="image" src="https://github.com/user-attachments/assets/23c709b1-9385-4787-9670-15bbc52bed42">

...and like the other region boxes, that box will be visible on any OP3-measured show if the sum of all regions in that area is over 3 percent.  Here's an example from [Think Fast, Talk Smart](https://op3.dev/show/ac5f85b4b76a4b558af0a3a6e9a99cba)
<img width="366" alt="image" src="https://github.com/user-attachments/assets/6a398ac6-7075-495b-9169-ab2e9a3c3498">

**Comment by NewbiePodcaster** [comment]
You are the best John! Thank you - I see it and will
keep that in mind for other regions.

Enjoy your Sunday.

Lola

**Comment by NewbiePodcaster** [comment]
Hello John,

I had a question regarding the data - I just want to make sure I am reading
these correctly.
- I have some episodes that have passed the 30-day mark, but it's not
reflected on the chart, does that mean there were no more downloads for
that episode? Examples would be 9/10 and 9/17. The former is over 30 days
and the later is at 29 days (as of today) but their charts end at day 22
and day 19 respectively.
- Also is there a way to download the unique listeners data? I may have
missed the option on the site.
- For the ID hash, you mentioned it is rotated for privacy, does that mean
a unique listener could be counted multiple times if their ID hash changes
every month? Or how do you adjust for that?

Thank you so much for your help.

Lola

**Reply by johnspurlock-skymethod** [reply]
Let's see:

> does that mean there were no more downloads for that episode

Yes, for example the last non-bot download for the 9/10 episode was on 10/2.  You can always see every download into a spreadsheet for a given month using the export box at the bottom of the stats page.

> is there a way to download the unique listeners data?

In that same export spreadsheet, you can use the `audienceId` column as a proxy for a unique listener, defined by the IAB as IP address (or IP address secure hash in OP3's case) + the user-agent (ie the app).  Note that this is a secure one way hash, and the key used is rotated every month, so think of an `audienceId` as only being valid within a given month.

> For the ID hash, you mentioned it is rotated for privacy, does that mean a unique listener could be counted multiple times

You'll notice that OP3 shows unique listeners for a given month only.  Since they all reset at the first of the month, you can't make any sums of unique listeners across month boundaries.  Downloads you can sum across months, but not unique listeners.

Taking a step back, the reason OP3 rotates the IP address secure hash (and thus the `audienceId`, which is based on it) every month, is so that callers can't take a really long view of data at a fixed IP (say at home) and start to figure out who it might be using months and months of data.  The only thing we lose when we do this is the ability to track these listeners across months, which is not the end of the world from a stats point of view, and seems like a reasonable tradeoff to maintain OP3's strong listener privacy stance.

**Comment by NewbiePodcaster** [comment]
Good morning John,

Thanks for taking the time to explain - this helps a lot.

I didn’t think of bots 😂- definitely don’t want their data.

So the 30 days count is a better measure, looks like OP3 is more accurate
than Simplecast (or others), is that correct?

And totally agree regarding listener privacy, only other question I had was
how do I get the ‘listen rate’ box show up? Saw the layout on Podnews’
podcast.

Thanks,
Lola


Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
In general, the 30-day episode download number is a measurement that several companies use as the best metric to compare shows.  If I were to add another "big box" at the top of the OP3 stats page for every show, it would be median 30-day episode downloads.  That really gives you a good idea of "how many regular listeners do I have?"

I won't impugn any other system, but I'll just say OP3 puts considerable effort into being as accurate as a prefix can be.  The gory details, if you are interested, are in the "How OP3 calculates downloads" link on every stats page.

OP3 (and every prefix and your hosting company) measures downloads on the server-side.  The _client-side_ Listen Time metric you'll see on some OP3 shows is from a very small number of special apps that have listeners that send micropayments as they are listening, which OP3 can also see if the podcaster chooses.  See the 2024-01-10 entry on the [Releases](https://op3.dev/releases) page for more info.

If you are not already doing so, you should log into [Apple Podcasts Connect](https://podcastsconnect.apple.com/) and periodically monitor the client-side stats they provide for your show.  Unfortunately they do not allow third-party integrations of that data, so you have to go there directly.

**Comment by NewbiePodcaster** [comment]
Hey John,

Hope you have been well. I am changing distribution hosts and I wanted to
see if there’s a best practice for removing the prefix from my old host and
just adding it to the new host.

Not sure if that will affect any dada but thought I’d do it end of the
month.

Thanks.
Lola

Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
Hi there - you shouldn't need to remove the prefix from your old host, just add it to your new host (if they support third-party prefixes, not all hosts do)

Once you've migrated, your old host should forward your old podcast RSS feed to a new location at the new host.  OP3 should notice this and associate listens at the new host with your existing show stats page.

Hope that helps!

**Comment by NewbiePodcaster** [comment]
Thanks John.

It currently shows an error on OP3 but hoping that gets fixed by tomorrow.

Will I lose the initial stats?
Lola

**Reply by johnspurlock-skymethod** [reply]
Please use this link going forward to access your show stats: https://op3.dev/show/cddc5343cf694cb099bf724d213b2ec6

That url uses the stable OP3 internal identifier and won't change even if your show changes hosts.  I see your new Captivate feed on the backend, the system should pick it up tomorrow, and allocate any downloads already occurring there.  I'll keep an eye on it.

**Comment by NewbiePodcaster** [comment]
Thank you John! I see my stats with the new link.

Lola

**Comment by NewbiePodcaster** [comment]
Hi John,

The African stars aren’t showing again for some reason. Can you please
help?

Thank you,
Lola

**Reply by johnspurlock-skymethod** [reply]
Let's see, are you referring to this show? https://op3.dev/show/cddc5343cf694cb099bf724d213b2ec6

I just took a look at the downloads (using the Export at the bottom of the page) and it looks like there were no downloads from Africa in February so far, and only one download in the entire month of January.  That box will only appear if there are some non-trivial percentage of downloads from there.

**Comment by NewbiePodcaster** [comment]
Yes - ok, I thought so but then I saw South Africa on the list of countries
in February which got me curious.

Lola

Regards, Lola Sent from Gmail Mobile

**Comment by NewbiePodcaster** [comment]
Read that wrong, South Africa was January but the African tab wasn’t
triggered.

Regards, Lola Sent from Gmail Mobile

**Comment by NewbiePodcaster** [comment]
Hi John,

Hope you had a good weekend.

Is there any way you still have access to my old data? I was trying to see
if at the very least I could get the October report for Top Countries and
the US Metro report from August to November.

Podcast - The Kadara woman Podcast

I didn’t realize at the time that the platform keeps a rolling 3-months.

Thanks,
Lola

Regards, Lola Sent from Gmail Mobile

**Comment by NewbiePodcaster** [comment]
Hi John,

Just bumping this to the top of your inbox. Thanks.

Lola

**Comment by johnspurlock-skymethod** [comment]
Hi there - yes the data for any OP3-measured show is never trimmed, like other services, it remains available for as long as you use the prefix, accessible using the [OP3 API](https://op3.dev/api/docs).

However, I suspect you might be talking about the hosted show stats page?  I think that may limit the view to the last three months.

**Comment by NewbiePodcaster** [comment]
Hi John,

Yes, asking about show stats page. I was curious if there was a way to
access past data. More of a one- off request.

Regards, Lola Sent from Gmail Mobile

**Reply by johnspurlock-skymethod** [reply]
Send me an email to john@op3.dev and let me know more details, this will help me keep track of it better
